src/ui/public/state_management/__tests__/config_provider.js

```diff
@@ -0,0 +1,32 @@
+import expect from 'expect.js';
+import ngMock from 'ng_mock';
+import '../config_provider';
+
+describe('State Management Config', function () {
+  let stateManagementConfig;
+
+  describe('is enabled', () => {
+    beforeEach(ngMock.module('kibana'));
+    beforeEach(ngMock.inject(function (_stateManagementConfig_) {
+      stateManagementConfig = _stateManagementConfig_;
+    }));
+
+    it('should be enabled by default', () => {
```

Comment:
__optional nit:__ in some places within this file you use `() => {}` (for `describe` and `it`) notation and in some `function() {}` (for top-level `describe` and `beforeEach`), can we be consistent on that? :)

Reply:
We can, it's a mix of old and new tests really. I'll update the existing tests to arrow functions so they're all the same.